### ISSUE TITLE
Documentation needs to say that refreshAuthLogic needs to return the axios instance

### ISSUE THREAD

**Issue by thomergil** [Flyrell/axios-auth-refresh#99]
Thank for you this package. Very useful. First of all, having to use `skipWhileRefreshing: false` is required for the desired behavior of the package, and I'd urge you to make this the default behavior. Other people seem to have suggested this already, and you have promised to make it so. Thank you for that. Please make it happen. 👍 

Secondly, documentation is unclear. It needs to mention that `refreshAuthLogic` needs to return the axios instance:

```
const refreshAuthLogic = (failedRequest: any): Promise<any> => {
  return axios.post(...).then(...);
}
```

The `return` is missing from the documentation and leads to errors.

**Comment by Flyrell** [comment]
Hi @thomergil, thanks for submitting the issue.

Yes I'm aware that `skipWhileRefreshing: false` is required and I plan to change it as soon as possible in the v3 of the library. 

Please, understand this is something I do in my free time while I'm working on other projects/libraries. If you'd like to fast-forward the development process, please consider contributing if that's something you'd be able to do. I'd be more than happy to do a code review & accept any pull requests on this package.

When it comes to the documentation, there's no `return` statement because of how the arrow functions work. When only one statement is executed and there's no wrapping curly braces `{}` the statement return type will be returned automatically. Although it's correct, I agree that the function can be written in `{}` and returned so the people seeing it will actually realise that the return is necessary for the library to work.

**Comment by SerhiyRomanov** [comment]
I agree with @thomergil because I am faced with the same error. I resolved it only after reading this issue.

**Comment by vinurs** [comment]
@SerhiyRomanov me too

**Comment by Flyrell** [comment]
I plan to include this change in v3.0.0 [PR #105].

**Comment by Flyrell** [comment]
Done, merged and published in #111.
